strings-csharp/StringFormattableString/StringFormattableStringTests/FormattableStringsUnitTests.cs

```diff
@@ -0,0 +1,44 @@
+namespace StringFormattableStringTests
+{
+    [TestClass]
+    public class FormattableStringsUnitTests
+    {
+        private readonly FormattableStringsMethods _formattableStringMethods = new();
+    
+        [TestMethod]
+        public void GivenAStringObject_WhenStringOperationsApplied_VerifyAccurateResults()
+        {
+            var stringObj = "Jane";
+            var sampleString = _formattableStringMethods.StringExample();
+
+            Assert.AreEqual(stringObj.Count(), stringObj.Length);
+            Assert.IsFalse(stringObj.Equals(sampleString));
+            Assert.AreEqual(sampleString.CompareTo(stringObj), 1);
```

Comment:
As a whole, it is hard to grasp what we are actually testing in this unit test. It would probably make the tests clearer if the `FormattableStringMethods` methods were changed to take both a name and an age argument. Then in the tests we could verify that the returned result is the type expected and contains the arguments/values expected. Then we just need to update the benchmark to pass in the arguments using either the `Argument` or `ArgumentSource` attributes.